third_party/move/move-core/types/src/vm_status.rs

```diff
@@ -162,6 +168,15 @@ impl VMStatus {
         self.status_code().status_type()
     }
 
+    pub fn is_aggregator_error(&self) -> bool {
```

Comment:
Same, let's move it to aggregator extension?

Reply:
My idea was that Aggregator extension file deals with `PartialVMError`. And in terms of hierarchy of errors that we have currently, `VMStatus` is a level about `PartialVMError`. I felt it would be a little confusing if we have code in aggregator_extension to deal with `VMStatus`.

Reply:
Removed it. I shifted the `is_aggregator_error()` check inline in `aptos_vm.rs`.